src/creatures/players/cyclopedia/player_cyclopedia.hpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Canary - A free and open-source MMORPG server emulator
+ * Copyright (©) 2019-2024 OpenTibiaBR <opentibiabr@outlook.com>
+ * Repository: https://github.com/opentibiabr/canary
+ * License: https://github.com/opentibiabr/canary/blob/main/LICENSE
+ * Contributors: https://github.com/opentibiabr/canary/graphs/contributors
+ * Website: https://docs.opentibiabr.com/
+ */
+
+#pragma once
+
+#include "creatures/creatures_definitions.hpp"
+#include "enums/player_cyclopedia.hpp"
+
+class Player;
+class KV;
+
+struct Summary {
+	uint16_t m_preyWildcards = 0;
+	uint16_t m_instantRewards = 0;
+	uint16_t m_hirelings = 0;
+
+	Summary(uint16_t mPreyWildcards, uint16_t mInstantRewards, uint16_t mHirelings) :
+		m_preyWildcards(mPreyWildcards), m_instantRewards(mInstantRewards), m_hirelings(mHirelings) { }
+};
+
+class PlayerCyclopedia {
+public:
+	explicit PlayerCyclopedia(Player &player);
+
+	Summary getSummary() {
+		return { getAmount(Summary_t::PREY_CARDS),
```

Comment:
If you move the function body to the cpp, you can remove an include from this hpp and avoid circular inclusion